### ISSUE TITLE
Replace slack-utils with the slack package

### ISSUE THREAD

**Issue by captn3m0** [sdslabs/jinora#71]
The `slack-utils` package was written when there weren't many good clients for Slack available. However:

- Jinora is the only app using it
- It hasn't been maintained in quite some time
- There are known security issues with the dependencies it is using

I've been trying to do a new release, but mocha is causing some issues with tests that I don't want to bypass: https://github.com/captn3m0/slack-utils/pull/7

But even then, this would be a stop-gap measure, since `slack-utils` relies on Slack Legacy Tokens and not the new API. Using the new API would also give Jinora some super-powers like supporting multiple organizations easily with one-click installs.

**Comment by vrongmeal** [comment]
Can you link me to the slack package you're talking about?

**Comment by captn3m0** [comment]
captn3m0/slack-utils on GitHub, slack-utils on NPM.

**Comment by vrongmeal** [comment]
Yeah... So I was asking about the package we want to replace slack-utils with.

**Comment by captn3m0** [comment]
Sorry, was on mobile.

https://www.npmjs.com/package/slack seems to support everything

**Comment by vrongmeal** [comment]
Thanks! I'll take a look at it :)

**Comment by captn3m0** [comment]
@vrongmeal Any update on this?

**Comment by vrongmeal** [comment]
Hi! Sorry I've been a bit busy. Totally forgot about this. I'll ask someone else if anyone is interested in changing the dependency.

**Comment by captn3m0** [comment]
Please do. I'm planning to delete the npm package by month end (31st July)

**Comment by captn3m0** [comment]
@vrongmeal Is someone picking this up?

**Comment by vrongmeal** [comment]
Yeah. @AyanChoudhary has taken this up.
